tripal_chado/src/Plugin/Field/FieldWidget/ChadoLocalSourceDataWidgetDefault.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Drupal\tripal_chado\Plugin\Field\FieldWidget;
+
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\tripal_chado\TripalField\ChadoWidgetBase;
+
+/**
+ * Plugin implementation of default Chado Sequence widget.
+ *
+ * @FieldWidget(
+ *   id = "chado_local_source_data_widget_default",
+ *   label = @Translation("Chado Local Source Data Widget Default"),
+ *   description = @Translation("The default chado local source data widget which allows curators to manually enter analysis source on the content edit page."),
+ *   field_types = {
+ *     "chado_local_source_data_default"
+ *   }
+ * )
+ */
+class ChadoLocalSourceDataWidgetDefault extends ChadoWidgetBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function formElement(FieldItemListInterface $items, $delta, array $element, array &$form, FormStateInterface $form_state) {
+
+    $item_vals = $items[$delta]->getValue();
+
+    $elements = [];
+    $elements['record_id'] = [
+      '#type' => 'value',
+      '#default_value' => $item_vals['record_id'] ?? 0,
+    ];
+    $elements['sourceuri'] =  [
+      '#title' => "Local Source URI",
```

Comment:
```suggestion
      '#title' => "Data Source URI",
```

Reply:
These were changed to "Name", "Version", "URI"